tests/sundials/n_vector.cc

```diff
@@ -33,6 +33,12 @@
 
 using namespace SUNDIALS::internal;
 
+// For better compatibility with older versions of SUNDIALS just define this as
+// a static member so we can use it whenever we need it
+#if DEAL_II_SUNDIALS_VERSION_GTE(6, 0, 0)
+static SUNContext nvector_context;
```

Comment:
Could you call this `global_nvector_context` to make its intent clear? Several of the SUNDIALS solvers have their own local context objects already, so the name would differentiate these more clearly.

Reply:
Sure.